### PR TITLE
Add solveset, linsolve, and linear_eq_to_matrix from solveset to __init__.py

### PR DESCRIPTION
**Pull request by asmeurer**
See #10006.

### CODE REVIEW

**Comment by aktech** [comment]
+1

File: sympy/solvers/solveset.py [hunk 1]
```diff
@@ -1139,9 +1137,7 @@ def linsolve(system, *symbols):
     Examples
     ========
 
-    >>> from sympy.solvers.solveset import linsolve
-    >>> from sympy import Matrix, S
-    >>> from sympy import symbols
+    >>> from sympy import Matrix, S, linsolve, symbols
```
Comment (aktech) [inline sympy/solvers/solveset.py]: BTW, `S` is not required.

Comment (asmeurer) [inline sympy/solvers/solveset.py]: It's used here https://github.com/sympy/sympy/pull/10049/files#diff-eec0422923e8f100745c015cd8fdd6cfR1170

Comment (aktech) [inline sympy/solvers/solveset.py]: Ah! I see.

**Pull request merged**